docs/logs/customizing-the-sdk/README.md

```diff
@@ -0,0 +1,81 @@
+# Customizing OpenTelemetry .NET SDK for Logs
+
+## OpenTelemetryLoggerProvider
+
+TODO
+
+## Building the OpenTelemetryLoggerProvider
+
+TODO
+
+## OpenTelemetryLoggerProvider configuration
+
+TODO
+
+### IncludeScopes
+
+TODO
+
+### IncludeFormattedMessage
+
+TODO
+
+### ParseStateValues
+
+TODO
+
+### AddProcessor
+
+TODO
+
+### SetResourceBuilder
+
+TODO
+
+## Filtering LogLevels
+
+`ILogger` implementations have a built-in mechanism to apply log filtering.
+This filtering lets you define the minimum
+[`LogLevel`](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.loglevel)
+for logs that are sent to each registered provider,
+including the `OpenTelemetryLoggerProvider`.
+You can use the filtering either in configuration (i.e. appsettings.json) or in code.
+
+### via appsettings.json
```

Comment:
I'd suggest to remove this whole section, and show only the code based one.
(appsettings.json require other things (likely DI or Host etc..), and we are not showing it here.)

Reply:
I didn’t want to leave this out because the ILogger docs state that this is the preferred method for configuration. 

Do you have any suggestions for documenting the alias? The class name doesn’t work here.

Reply:
OpenTelemetry is the alias for `OpenTelemetryLoggerProvider`, that may be used in configuring filtering rules.